### ISSUE TITLE
Just crashes, do I do something wrong?

### ISSUE THREAD

**Issue by remyvanlis** [Jazzmoon/btd6_autoplay#10]
After a long hour of debugging why tesseract wouldn't work (was my fault, luckily) everything seemed to work. Opening the run.bat shows a happy console and allows me to type in "darkcastle" and "easy". After that, it appears to be trying to start - yet all it does is crash as soon as I open the heroes selection menu in-game. If I do nothing, the program also does nothing, and once I select the hero selection menu, it shows me this:
![image](https://user-images.githubusercontent.com/62595579/168931188-b666cdae-396f-47e4-811f-7542254272cb.png)

Anything I probably did wrong?

**Comment by AstroCoding** [comment]
Are you attempting to run the script from menu? If so, it will never work. You must have launched the map already. From there, you start the script for the map you are currently in and it should begin if configured.

**Comment by remyvanlis** [comment]
I see, thanks for notifying me on that one. However, I still receive the same error:

![image](https://user-images.githubusercontent.com/62595579/168991204-f5536782-a67f-4732-b007-da55e13b2355.png)


I tried changing the index on line 61 from [1] to [0], breaking the progress bar, just so it won't die at least. That doesn't help either, not really sure what I'm still doing wrong here

**Comment by remyvanlis** [comment]
Alright, so it started moving the mouse once I changed my resolution to 1080p. It's not even getting close to actually placing something though, what is the desired resolution?

**Comment by remyvanlis** [comment]
well, been busy for a bit. Apparently pygui can't press buttons on windows 10 and 11 (at least, not for me and at least one other person) - so we replaced it. Also thought it was fun to add a timer and something that allows leveling up, not even sure if it really works since we hardly ever level up but whatever:p also fixed some clicking coords, it didn't restart after winning.

it appears I can't push a branch, so here's the new sourcecode, do with it what you want:
https://drive.google.com/file/d/1uN026PH1pxzw_twjvkb66QJ8OzuPdYzb/view?usp=sharing

**Comment by AstroCoding** [comment]
@remyvanlis This project was a personal project and not designed for public consumption. The issues you are having appear to be issues with Pytesseract's ability to parse the round. We ran the game at 1920x1080 full screen, and tweaked every value to be the exact values we needed to run it on our personal machine, but we are aware this won't work for everyone.
Sadly, I don't have a fix for you. Regretfully, we don't have the time to continue development.
